web/src/main/java/com/epam/esm/controller/CertificateController.java

```diff
@@ -0,0 +1,105 @@
+package com.epam.esm.controller;
+
+import com.epam.esm.dto.CertificateDto;
+import com.epam.esm.entity.Certificate;
+import com.epam.esm.exception.InvalidEntityException;
+import com.epam.esm.repository.SearchCriteria;
+import com.epam.esm.service.CertificateService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+/**
+ * Class containing public REST API endpoints related to {@link Certificate} entity.
+ *
+ * @author Egor Geller
+ */
+@RestController
+@RequestMapping("/api/certificates")
```

Comment:
Read about versioning

Reply:
https://stackoverflow.com/questions/389169/best-practices-for-api-versioning

Reply:
I read about versioning, did not conclude an opinion on which type to use,  just placing RequestMapping("api/{ver}/..")?